src/test/suite/connectionController.test.ts

```diff
@@ -512,7 +512,7 @@ suite('Connection Controller Test Suite', function () {
 
   test('"getConnectionStringFromConnectionId" returns the driver uri of a connection', async () => {
     const expectedDriverUri =
-      'mongodb://localhost:27018/?readPreference=primary&appname=mongodb-vscode%200.5.1-dev.0&directConnection=true&ssl=false';
```

Comment:
Can we read the version from package.json? A separate PR is also ok if you want to go with this!

Reply:
Yeah let's grab in a separate pr hah